### PR TITLE
feat: 文件管理编辑文件增加左侧目录树

### LINKED ISSUE #3624: [FEATURE]文件编辑器功能的建议
### 1Panel 版本

v1.9.3

### 请描述您的需求或者改进建议

建议可以窗口最大化和左侧窗口选择目录文件进行多个编辑，例如宝塔的那样很方便实用。

### 请描述你建议的实现方案

_No response_

### 附加信息

_No response_

### PR DESCRIPTION
**Pull request by lan-yonghui**
Refs: #3624

#### Please indicate you've done the following:

- [ ] Made sure tests are passing and test coverage is added if needed.
- [ ] Made sure commit message follow the rule of [Conventional Commits specification](https://www.conventionalcommits.org/).
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed.

### CODE REVIEW

File: backend/app/service/file.go [hunk 1]
```diff
@@ -100,27 +105,79 @@ func (f *FileService) SearchUploadWithPage(req request.SearchUploadWithPage) (in
...
+	}
+	return nil
+}
+
+func (f *FileService) buildChildNode(childNode *response.FileTree, fileInfo *files.FileInfo, op request.FileOption, level int) error {
+	op.Path = fileInfo.Path
+	subInfo, err := files.NewFileInfo(op.FileOption)
+	if err != nil {
+		if os.IsPermission(err) || errors.Is(err, unix.EACCES) {
+			fmt.Printf("File Tree: Skipping %s due to permission denied\n", fileInfo.Path)
```
Comment (zhengkunwang223) [inline backend/app/service/file.go]: 这里可以用 global.LOG.Info 打印日志

Comment (lan-yonghui) [reply backend/app/service/file.go]: nice

**Review by wanghe-fit2cloud — COMMENTED** [review]
/lgtm

**Pull request merged**